packages/api/core/src/util/index.ts

```diff
@@ -24,9 +24,7 @@ export default class ForgeUtils {
 
   getElectronVersion = getElectronVersion;
 
-  hasYarn = hasYarn;
-
-  yarnOrNpmSpawn = yarnOrNpmSpawn;
+  spawnPackageManager = spawnPackageManager;
```

Comment:
Is a change in API signatures for `@electron-forge/core-utils` a breaking change? If it is, I can add a temporary shim for these utils and deprecate them.